todo_app/templates/index.html

```diff
@@ -10,8 +10,17 @@ <h1 class="display-4">To-Do App</h1>
   <div class="row">
     <div class="col-md-12">
       <h2>Items</h2>
+      <form action = '/' method = "POST">
+        <input type = "text" name = "user todo" >  </input>
```

Comment:
This works nicely, but I'll admit I never quite trust spaces in key fields to be rendered correctly (e.g. not to be encoded) so personally I'd switch this `name` to use an underscore:
```suggestion
        <input type = "text" name = "user_todo" >  </input>
```
(Bonus reason - since we need this to be exact, this is a bit easier to spot when there are subtle differences, such as two spaces or two underscores from a typo, or e.g. a tab sneaks in!)

Reply:
(Note that if we do change this, we should also change the `app.py` version too!)